src/N3Writer.js

```diff
@@ -84,8 +85,11 @@ export default class N3Writer {
   // ### `_writeQuad` writes the quad to the output stream
   _writeQuad(subject, predicate, object, graph, done) {
     try {
+      if (this._graphs === 'error' && !DEFAULTGRAPH.equals(graph)) {
+        done(new Error('Encountered graph name, this is forbidden.'));
```

Comment:
```suggestion
        throw new Error('The chosen serialization settings do not support triples in a non-default graph.');
```

Have to throw here; done doesn't exit the function.